src/app/WriteHandler.h

```diff
@@ -36,13 +36,13 @@ namespace app {
 /**
  *  @brief The write handler is responsible for processing a write request and sending a write reply.
  */
-class WriteHandler
+class WriteHandler : public Messaging::ExchangeDelegate
```

Comment:
yep, we need this to handle incoming chunking message

Reply:
so this logic can only be exercised after @erjiaqing chunking write get landed.

Reply:
Odd - that logic somehow got removed when I did some conflict resolutions. It's back in now.